MapboxNavigation/Geometry.swift

```diff
@@ -1,41 +1,40 @@
 import CoreLocation
 import MapboxDirections
 
-typealias LocationRadians = Double
-typealias RadianDistance = Double
-typealias RadianDirection = Double
+public typealias LocationRadians = Double
+public typealias RadianDistance = Double
+public typealias RadianDirection = Double
+let metersPerRadian = 6_373_000.0
 
-extension CLLocationDegrees {
-    func toRadians() -> LocationRadians {
+public extension CLLocationDegrees {
+    public func mb_toRadians() -> LocationRadians {
         return self * M_PI / 180.0
     }
     
-    func toDegrees() -> CLLocationDirection {
+    public func mb_toDegrees() -> CLLocationDirection {
         return self * 180.0 / M_PI
     }
 }
 
-let metersPerRadian = 6_373_000.0
-
-struct RadianCoordinate2D {
+public struct RadianCoordinate2D {
     var latitude: LocationRadians
     var longitude: LocationRadians
     
-    init(latitude: LocationRadians, longitude: LocationRadians) {
+    public init(latitude: LocationRadians, longitude: LocationRadians) {
         self.latitude = latitude
         self.longitude = longitude
     }
     
-    init(_ degreeCoordinate: CLLocationCoordinate2D) {
-        latitude = degreeCoordinate.latitude.toRadians()
-        longitude = degreeCoordinate.longitude.toRadians()
+    public init(_ degreeCoordinate: CLLocationCoordinate2D) {
+        latitude = degreeCoordinate.latitude.mb_toRadians()
+        longitude = degreeCoordinate.longitude.mb_toRadians()
     }
     
     
     /*
      Returns direction given two coordinates.
     */
-    func direction(to coordinate: RadianCoordinate2D) -> RadianDirection {
+    public func MBDirection(to coordinate: RadianCoordinate2D) -> RadianDirection {
```

Comment:
This will either bridge to Objective-C as `MBDirection()` or `MBDirectionWithTo()`. Either way would be terrible. The solution is to define `MBDirectionToRadianCoordinate2D()` in Objective-C, mark it with `NS_SWIFT_NAME` or `NS_REFINED_FOR_SWIFT`, and implement it in Swift.

The same goes for all the other standalone functions you want to make public. That’s pretty annoying, which is why I think we should try to publicize as few functions as possible for your needs.